# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1314729260 14400
#      Tue Aug 30 14:34:20 2011 -0400
# Branch stable
# Node ID 9b191d0e547f9a760d9d046b7ae16cf1ddc3877d
# Parent  58d704d3283c9c7951cc9f2ac1f5b93963f14b7f
doc: fix typo in ndims doc string

* data.cc (Fndims): Fix typo in doc string.

diff --git a/src/data.cc b/src/data.cc
--- a/src/data.cc
+++ b/src/data.cc
@@ -2024,17 +2024,17 @@ DEFUN (ndims, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} ndims (@var{a})\n\
 Return the number of dimensions of @var{a}.\n\
 For any array, the result will always be larger than or equal to 2.\n\
 Trailing singleton dimensions are not counted.\n\
 \n\
 @example\n\
 @group\n\
-  ndims (ones (4, 1, 2, 1)\n\
+  ndims (ones (4, 1, 2, 1))\n\
      @result{} 3\n\
 @end group\n\
 @end example\n\
 @end deftypefn")
 {
   octave_value retval;
 
   if (args.length () == 1)
